### ISSUE TITLE
Not working with PIP install

### ISSUE THREAD

**Issue by Dlutz159** [hSaria/ChromaTerm#77]
## Description
When installing with PIP I was unable to use 'ct' from the command line. Attempts to use it resulted in a command not found error.
Not sure if the PATH wasn't updated or where it went wrong. 

Using cloning the git Repo and using CT from it works fine.

## Environment
-   OS: Pop!_OS 19.10
-   ChromaTerm: Current as of 11/4/19

## Configuration
Default CT config

##Other
Running CT this way is fine with me. (git clone vs PIP) Just wanted to let you know that I had this issue installing with PIP since it is the suggested install method. 
Love the project!

**Comment by hSaria** [comment]
That's the first time I've seen this issue. Would you mind running the following and attaching the output? Considering I'm using the native `scripts` directive in `setup.py` to install the binary, I'm gonna guess this will affect other packages installed with pip, too.
```
echo $PATH
pip3 --version
pip3 install chromaterm --force-reinstall --verbose
pip3 show chromaterm --files
```

> There shouldn't be any sensitive information in the output, but feel free to anonymize it.

**Comment by hSaria** [comment]
TL;DR, the solution is adding this to your `.bash_profile` or `.bashrc` (or the config file of the shell you use):
```
PATH="$HOME/.local/bin:$PATH"
```

So I installed Pop!_OS and found where the problem lies.

When installing a module via pip from your user, it adds the package to `~/.local/` as it doesn't have the permission to write to the system-wide location. Thus, the binary ends up in `~/.local/bin/` which is not in your `$PATH` by default. The solution above fixes that.

As you might've guessed by now, installing the package as root (i.e. `sudo pip3 install chromaterm`) will put the binary in `/usr/local/bin/` which is already a part of your `$PATH`. If you don't you want to modify your `$PATH`, install the module as `root`.
